operators/pkg/instmetrics/metrics_scraper.go

```diff
@@ -84,7 +84,7 @@ func (ms *MetricsScraper) Start(ctx context.Context) {
 		forge.XVncName,
 		forge.WebsockifyName,
 		forge.ContentDownloaderName,
-		forge.MyDriveName,
+		forge.FileBrowserName,
```

Comment:
Could be dropped, since you removed that container.

Reply:
Done